packages/nimbus/src/components/tooltip/tooltip.mdx

```diff
@@ -13,36 +11,123 @@ menu:
 tags:
   - component
   - media
+figmaLink: >-
+  https://www.figma.com/design/gHbAJGfcrCv7f2bgzUQgHq/NIMBUS-Guidelines?node-id=1695-45519&m
 ---
 
-# Tooltip
+# Tooltips
+
+---
+
+Tooltips display informative text when users hover over or focus on an element.
+
+## Overview
+
+Tooltips offer concise, non-obtrusive guidance and supplemental information, enhancing user understanding and interaction with UI elements.
+
+### Resources
+
+Deep dive on details and access design library.
+
+[TECH RELATED LINK](https://example.com)
+[TECH RELATED LINK](https://example.com)
+[Figma library](https://www.figma.com/design/gHbAJGfcrCv7f2bgzUQgHq/NIMBUS-Guidelines?node-id=1695-45519&m)
+
+## Variables
+
+Get familiar with the features. 
+
+### Placements
+
+Tooltip placements can populate in different areas, please chose a direction that will be most visible for the users. and the elements that may be interacting around them. Options include bottom left, bottom center, bottom right, top left, top center, top right, left top, left center, left bottom, right top, right center, right bottom. 
+
+## Guidelines
+
+---
+
+Use tooltips to provide brief explanations, definitions, or instructions for interactive elements, enhancing the user experience.
```

Comment:
I know the header dividers are in Figma, but they aren't consistent for each header, so let's remove it here.